src/v2/Components/ArtworkFilter/Utils/urlBuilder.tsx

```diff
@@ -32,8 +32,14 @@ export const paramsToCamelCase = params => {
   }, {})
 }
 
-export const buildUrl = (state: ArtworkFilters, pathname?: string): string => {
-  const params = removeDefaultValues(state)
+export const buildUrl = (
+  state: ArtworkFilters,
+  {
+    pathname,
+    defaultValues,
+  }: { pathname?: string; defaultValues?: Partial<ArtworkFilters> }
```

Comment:
(non-blocking) - since the second arg is expanding into something more options centric it might be helpful to describe it like so:

```jsx
const buildUrl = (state: ArtworkState, options: BuildUrlOptions) => { ... }
```